src/rofi_rbw/rofi_rbw.py

```diff
@@ -92,7 +92,9 @@ def __parse_arguments(self) -> argparse.Namespace:
         parser.add_argument(
             "--no-folder", dest="show_folders", action="store_false", help="Don't show the entry's folder"
         )
+        parser.add_argument("--no-cache", dest="use_cache", action="store_false", help="Don't save history in cache")
```

Comment:
I'd like to keep the argument and the default together, like this:
```
parser.add_argument("--no-folder"...)
parser.set_defaults(show_folders=True)
```
Could you please move them accordingly?